### PR TITLE
Added event page for ELIXIR Proteomics Community talk

### PR DESCRIPTION
**Pull request by ewels**
Also fixed up the readme file a little (new laptop and I'm trying not to install MAMP, so was working through the docker instructions myself).

### CODE REVIEW

**Comment by ewels** [comment]
I couldn't figure out why the event metadata wasn't rendering in the header, then I realised that this was a minor bug that had found its way through after the refactoring of the `parse_md()` function.

Fixed this, plus added a couple of minor tweaks to the output.

File: public_html/events.php [hunk 1]
```diff
@@ -51,14 +51,19 @@ function sanitise_date_meta($event){
...
     $colour_class = $event_type_classes[strtolower($meta['type'])];
     $icon_class = $event_type_icons[strtolower($meta['type'])];
     $subtitle = '<span class="badge badge-'.$colour_class.' mr-3"><i class="'.$icon_class.' mr-1"></i>'.ucfirst($meta['type']).'</span> '.$subtitle;
   }
 
-  $event = sanitise_date_meta($meta);
+  $event = sanitise_date_meta($output["meta"]);
+  print_r($meta);
+  echo var_dump($meta);
+  echo "HELLLOOWOOWOW";
```
Comment (mashehu) [inline public_html/events.php:66]:
```suggestion
 
```

**Review by mashehu — CHANGES_REQUESTED** [review]

**Comment by mashehu** [comment]
One small change needed, rest looks good. 

Thanks for the mention in the credits 🙂

File: public_html/events.php (hunk 1 above)
Comment (ewels) [reply public_html/events.php:66]: Hah, oops! Well spotted 👀 😆

**Pull request merged**